src/main/java/org/generationcp/middleware/domain/germplasm/GermplasmUpdateDTO.java

```diff
@@ -0,0 +1,108 @@
+package org.generationcp.middleware.domain.germplasm;
+
+import org.pojomatic.Pojomatic;
+import org.pojomatic.annotations.AutoProperty;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@AutoProperty
+public class GermplasmUpdateDTO {
+
+	private Integer gid;
+	private String germplasmUUID;
+	private String preferredName;
```

Comment:
We should name this as `preferredNameType`. It's confusing to use `preferredName` as its value is not the actual name value but the name type.